### DISCUSSION TITLE
請問PyCharm無法安裝最新版

### DISCUSSION THREAD

**Discussion by Marniew** [bluelovers/idea-l10n-zht#14] [Q&A] [answered]
![PyCharm無法安裝](https://user-images.githubusercontent.com/76925613/160273022-d254a0d5-1e54-45c9-90a9-2a8eeafec83b.png)
安裝221版會變這樣↑
目前只能安裝213.284版
有大大能解決嗎？

**Comment by bluelovers** [comment, marked as answer]
這應該是因為你沒有安裝最新版的 PyCharm 
如果仍然還在使用舊版或非EAP/BETA版的話

請參考使用
https://github.com/bluelovers/idea-l10n-zht#other-releases

透過 以下指令 來打包 具有最新版翻譯轉換的 213 語言包
yarn run plugin:series:build

因為 221 版本的語言包檔案跟 213 版本會在某些地方不同
(雖然大多數情況下都是一樣的就是
